components/RelatedReplies.js

```diff
@@ -1,12 +1,57 @@
 import React from 'react';
+import { t } from 'ttag';
+import { makeStyles } from '@material-ui/core/styles';
 import gql from 'graphql-tag';
 
 import { TYPE_NAME, TYPE_DESC } from 'constants/replyType';
 import ExpandableText from './ExpandableText';
-import { format, formatDistanceToNow } from 'lib/dateWithLocale';
 import { linkify, nl2br } from 'lib/text';
 import Link from 'next/link';
-import { sectionStyle } from './ReplyConnection.styles';
+import PlainList from 'components/PlainList';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    position: 'relative',
+    margin: '6px 0',
+    border: `1px solid ${theme.palette.secondary[100]}`,
+    borderRadius: 8,
+  },
+  body: {
+    padding: 24,
+  },
+  link: {
+    position: 'absolute',
+    right: 0,
+    top: 0,
+    padding: 8,
+    borderBottomLeftRadius: 8,
+    borderLeft: `1px solid ${theme.palette.secondary[100]}`,
+    borderBottom: `1px solid ${theme.palette.secondary[100]}`,
+    textDecoration: 'none',
+    color: theme.palette.secondary[300],
+  },
+  blockquote: {
+    fontSize: 13,
+    color: theme.palette.secondary[200],
+    borderLeft: `2px solid ${theme.palette.secondary[200]}`,
+    paddingLeft: 8,
+    marginLeft: 0,
+  },
+  title: {
+    marginBottom: '.5rem',
+  },
+  submit: {
+    borderBottomLeftRadius: 7,
+    borderBottomRightRadius: 7,
+    backgroundColor: theme.palette.primary[500],
+    color: theme.palette.common.white,
```

Comment:
I think that instead of customizing a button from browser style into primary button, we should use [Material UI Contained Buttons with `disableElevation`](https://material-ui.com/components/buttons/#contained-buttons). It has better handle over hover & active states as well.